### PR TITLE
Update CHANGELOG for Consul 1.10.0

### PR DESCRIPTION
**Pull request by thisisnotashwin**
- Update values file for licenseAutoloading to specify additional supported Consul versions for license autoloading.

How I expect reviewers to test this PR: Content review.

Checklist:
- [x] CHANGELOG entry added (*HashiCorp engineers only, community PRs should not add a changelog entry*)

### CODE REVIEW

**Review by david-yu — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,130 @@
...
+* Updated the default envoy image to `envoyproxy/envoy-alpine:v1.18.3`.
+* Add support to set the nodePort value in the Consul UI Service. [[GH-878](https://github.com/hashicorp/consul-helm/pull/878)]
+* Allow setting annotations on service accounts for: server, client, client snapshot agent, connect inject, controller, ingressGateways, meshGateway,
   syncCatalog, and terminatingGateways. [[GH-964](https://github.com/hashicorp/consul-helm/pull/964)]
 * Add support for `global.recursors` to set Consul's [`-recursor` flag](https://www.consul.io/docs/agent/options#_recursor) [[GH-985](https://github.com/hashicorp/consul-helm/pull/985)].
 * Delete secrets created by webhook-cert-manager when the deployment is deleted. [[GH-987](https://github.com/hashicorp/consul-helm/pull/987)]
-* Add support for `DialedDirectly` field in `ServiceDefaults` and `ProxyDefaults`. [[GH-992](https://github.com/hashicorp/consul-helm/pull/992)]
+* Specify `kubeVersion` in `Chart.yaml` to denote that this chart is compatible with Kubernetes 1.16+. [[GH-883](https://github.com/hashicorp/consul-helm/pull/883)]
```
Comment (lkysow) [inline CHANGELOG.md:28]: This should be in breaking changes and should read "Update supported `kubeVersion` in `Chart.yaml` from `>=1.13.0-0` to `>=1.16.0-0`.

**update:** I see you have that there already. I don't think you need this bullet.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,130 @@
...
+BREAKING CHANGES
+* [Enterprise] For versions of Consul Enterprise 1.9 <v1.9.7 and Consul Enterprise 1.8 < 1.8.12 , if the license was provided as a Kubernetes secret, the key `server.enterpriseLicense.enableLicenseAutoload`
+  needs to explicitly be set to `false`in order for the license job to run.
+```yaml
+server:
+  enterpriseLicense:
+    secretName: <name-of-kubernetes-secret>
+    secretKey: <name-of-key-whose-value-is-the-license>
+    enableLicenseAutoload: false
+```
```
Comment (lkysow) [inline CHANGELOG.md:39]:
```suggestion
* [Enterprise] For versions of Consul Enterprise 1.9 <v1.9.7 and Consul Enterprise 1.8 < 1.8.12 , if the license was provided as a Kubernetes secret, the key `server.enterpriseLicense.enableLicenseAutoload`
  needs to explicitly be set to `false`in order for the license job to run:
  ```yaml
  server:
    enterpriseLicense:
      secretName: <name-of-kubernetes-secret>
      secretKey: <name-of-key-whose-value-is-the-license>
      enableLicenseAutoload: false
  ```

  See [https://consul.io/docs/upgrading/instructions/upgrade-to-1-10-x](https://consul.io/docs/upgrading/instructions/upgrade-to-1-10-x) for more details.
```

**Review by kschoche — APPROVED** [review]
🔥

File: CHANGELOG.md (hunk 2 above)
Comment (lkysow) [reply CHANGELOG.md:39]: I think this suggestion got mangled, may need to change by hand rather than though "add suggestion"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,130 @@
...
+* Remove templates for a demo installation of Grafana in light of recent license changes to Grafana. [[GH-930](https://github.com/hashicorp/consul-helm/pull/930)]
```
Comment (lkysow) [inline CHANGELOG.md:40]:
```suggestion
* Remove templates for a demo installation of Grafana in light of recent license changes to Grafana's licensing. If you were previously setting `grafana.enabled` to `true` you must now install Grafana through their Helm chart (https://artifacthub.io/packages/helm/grafana/grafana). [[GH-930](https://github.com/hashicorp/consul-helm/pull/930)]
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,16 +1,130 @@
...
+* Remove support for `admissionregistration.k8s.io/v1beta1` version of MutatingWebhookConfiguration as it is deprecated in Kubernetes 1.16+. Only the `admissionregistration.k8s.io/v1`
+  version will be supported. [[GH-914](https://github.com/hashicorp/consul-helm/pull/914)]
+* Minimum Kubernetes versions supported is 1.16+. [[GH-883](https://github.com/hashicorp/consul-helm/pull/883)]
+* Connect: The Helm values for health checks and cleanup controller have been removed: `connectInject.healthChecks` and `connectInject.cleanupController`. [[GH-899](https://github.com/hashicorp/consul-helm/pull/899)]
```
Comment (lkysow) [inline CHANGELOG.md:44]:
```suggestion
* Connect: The Helm values for health checks and cleanup controller have been removed: `connectInject.healthChecks` and `connectInject.cleanupController`. This functionality is now enabled by default. You can delete these keys or leave them as they will no longer have any effect. [[GH-899](https://github.com/hashicorp/consul-helm/pull/899)]
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,16 +1,130 @@
...
+* Connect: connect webhook deployment now uses `webhook-cert-manager` to bootstrap the webhook certificates instead of generating them inside of the webhook. [[GH-861](https://github.com/hashicorp/consul-helm/pull/861)]
```
Comment (lkysow) [inline CHANGELOG.md:45]: Is this a breaking change?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,16 +1,130 @@
...
+* Connect: Kubernetes Services are now required for all connect injected applications.
```
Comment (lkysow) [inline CHANGELOG.md:46]: This should be at the top of the list

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,16 +1,130 @@
...
+  The Kubernetes service name will be used as the service name to register with Consul unless the annotation `consul.hashicorp.com/connect-service` is provided to the pod to override this.
+  If using ACLs the ServiceAccountName must match the service name used with Consul.
+  
+Example Service:
```
Comment (lkysow) [inline CHANGELOG.md:50]: Need to indent this so it's under the bullets

**Review by lkysow — COMMENTED** [review]
half-way through the review but need to drop for a bit so submitting now

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,16 +1,157 @@
 ## UNRELEASED
 
+FEATURES:
+* Connect: Support transparent proxy. [[GH-905](https://github.com/hashicorp/consul-helm/pull/905)]
+  This feature allows users to reach other services on the Consul Service Mesh by using KubeDNS instead of using `localhost` and enforces all inbound and outbound traffic within a pod to go through the Envoy proxy.
+  Please see [Transparent Proxy](https://www.consul.io/docs/connect/transparent-proxy ) docs for more information.
+
+  **Note: This feature requires consul-k8s `v0.26.0` or higher.**
```
Comment (lkysow) [inline CHANGELOG.md:8]:
```suggestion
  **Note: This feature requires Consul 1.10.0 or higher and consul-k8s `v0.26.0` or higher.**
```

File: CHANGELOG.md (hunk 5 above)
Comment (thisisnotashwin) [reply CHANGELOG.md:74]: i guess it is breaking in the sense that users who used to provide certificates for the webhook no longer can, but we did validate that it was broken. ill move it to improvements.

File: CHANGELOG.md (hunk 8 above)
Comment (lkysow) [reply CHANGELOG.md:8]: Do we have separate docs for tproxy we can link to as well?

**Review by lkysow — APPROVED** [review]
Great work corralling these.

File: CHANGELOG.md (hunk 8 above)
Comment (thisisnotashwin) [reply CHANGELOG.md:8]: Line 6 seems to link to the consul.io docs to tproxy..do you mean other docs?

Comment (lkysow) [reply CHANGELOG.md]: gotcha, nvm

**Pull request closed without merging**